### ISSUE TITLE
Container fails to build

### ISSUE THREAD

**Issue by wltechblog** [simonlui/Docker_IPEX_ComfyUI#4]
Fresh install on Ubuntu 22.04 with docker 24.0.5, build command fails at step 8.

```Step 8/59 : RUN apt-get update &&     apt-get install -y --no-install-recommends --fix-missing     intel-oneapi-runtime-dpcpp-cpp=${DPCPP_VER}     intel-oneapi-runtime-mkl=${MKL_VER}     intel-oneapi-runtime-openmp=${DPCPP_VER}     intel-oneapi-compiler-shared-common-${CMPLR_COMMON_VER}=${DPCPP_VER}
 ---> Running in ab57742100a0
Hit:1 http://security.ubuntu.com/ubuntu jammy-security InRelease
Hit:2 http://archive.ubuntu.com/ubuntu jammy InRelease
Hit:3 http://archive.ubuntu.com/ubuntu jammy-updates InRelease
Get:4 https://apt.repos.intel.com/oneapi all InRelease [5676 B]
Hit:5 http://archive.ubuntu.com/ubuntu jammy-backports InRelease
Get:6 https://apt.repos.intel.com/oneapi all/main all Packages [143 kB]
Get:7 https://apt.repos.intel.com/oneapi all/main amd64 Packages [502 kB]
Fetched 650 kB in 2s (432 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 intel-oneapi-runtime-dpcpp-cpp : Depends: intel-oneapi-runtime-compilers (= 2023.2.2-47) but it is not going to be installed
                                  Depends: intel-oneapi-runtime-dpcpp-cpp-common (= 2023.2.2-47) but 2023.2.4-49553 is to be installed
                                  Depends: intel-oneapi-runtime-opencl (= 2023.2.2-47) but 2023.2.4-49553 is to be installed
E: Unable to correct problems, you have held broken packages.
The command '/bin/sh -c apt-get update &&     apt-get install -y --no-install-recommends --fix-missing     intel-oneapi-runtime-dpcpp-cpp=${DPCPP_VER}     intel-oneapi-runtime-mkl=${MKL_VER}     intel-oneapi-runtime-openmp=${DPCPP_VER}     intel-oneapi-compiler-shared-common-${CMPLR_COMMON_VER}=${DPCPP_VER}' returned a non-zero code: 100

```

**Comment by simonlui** [comment]
It seems like Intel's packaging changed for whatever reason. I just tested and committed some changes to the repository that should address this. Can you confirm if you are able to get everything to run now?

**Comment by simonlui** [comment]
Hello, @wltechblog. Can you confirm if v0.1.0, which is the version that fixed your bug, addressed your issue? I have also moved onto v0.2.0pre for getting everything up to date unofficially with IPEX and other things for running ComfyUI, which may also help here for compatibility and speed reasons. I will be closing the issue if I don't hear back in a short amount of time.

**Comment by wltechblog** [comment]
Hello, sorry I no longer have access to the system with the ARC card and am unable to re-test.
